test/fixtures/rules/std_rule_cases/L019.yml

```diff
@@ -230,10 +230,10 @@ trailing_comma_move_past_several_comment_lines:
     global_actions_states
   configs:
     core:
-      # Set runaway_limit=1 to verify the fix only requires one pass. In an
+      # Set runaway_limit=2 to verify the fix only requires one pass. In an
       # earlier version, the comma before "SAFE_DIVIDE()" was being moved one
       # line per pass. Too lazy!
-      runaway_limit: 1
+      runaway_limit: 2
```

Comment:
Now that we treat hitting the limit as an error, these tests need one additional loop (First iteration generates a fix, then the second iteration generates **no** fixes and the loop exits). One iteration would've not only caused the linter to report an error, but also to discard the fixes!)